style.css

```diff
@@ -1,8 +1,144 @@
 body {
-  box-sizing: border-box;
   margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.nav__bar {
+  background: #64b5f6;
+  color: white;
+  overflow: auto;
+}
+
+.auth__name {
+  text-align: left;
+  margin-left: 2rem;
+}
+
+.links {
+  margin-right: 3rem;
+  text-align: right;
+  text-decoration: none;
+  color: white;
 
 }
-*, *:before, *:after {
-  box-sizing: inherit;
+
+.nav-right {
+  text-align: right;
+}
+
+.container {
+  /*grow up to 65rem*/
+  max-width: 70rem;
+  /*center if there is width left*/
+  margin: auto;
+}
+
+.card_left {
+  width: calc(100% / 2 - 1rem - 10rem);
+  box-shadow: .2rem .2rem .4rem;
+  float: left;
+}
+
+.card_right {
+  width: calc(100% / 2 - 1rem - 10rem);
+  box-shadow: .2rem .2rem .4rem;
+  float: right;
+}
+
+.post__title {
+  text-align: center;
+  background: #ffd740;
+  padding-top: 5rem;
+  padding-bottom: 5rem:
+}
+
+.post {
+  text-align: center;
+}
+
+.img {
+  max-width: 100%;
+  border-radius: 50%;
+  position: absolute;
+  z-index: auto;
+  top: 14.5%;
+  left: 3%;
+}
+
+.img_2 {
```

Comment:
I think what's happening is that you need to use `position: relative` on the card elements so that the cards AND their headers can be repeatable and the `position: absolute` of the images will be in reference to the card instead of the page.

This will also help keep things together when the page resizes.